packages/feedback/src/core/integration.ts

```diff
@@ -252,7 +252,7 @@ export const buildFeedbackIntegration = ({
 
     const _createActor = (optionOverrides: OverrideFeedbackConfiguration = {}): ActorComponent => {
       const shadow = _createShadow(_options);
-      const actor = Actor({ triggerLabel: _options.triggerLabel, shadow });
+      const actor = Actor({ triggerLabel: optionOverrides.triggerLabel || _options.triggerLabel, shadow });
```

Comment:
I would split and reorder `mergedOptions` below so that we have:
1) merged options of `_options` and `optionOverrides`
2) pass a new object of merged options + the actor callbacks to `_attachTo`

Then we can use the merged options for creating a shadow + actor so that we can avoid this type of issue in the future where the wrong options are used.